docs/porting/target/tickless/tickless.md

```diff
@@ -0,0 +1,64 @@
+# Tickless
+
+## Tickless mode
+
+Mbed OS has SysTick timer which generates tick in a periodic interval. The interrupt generated on every tick to reschedule the threads. The periodic tick configures as 1ms for Mbed OS.
+
+![](./resources/Tick.png)
+
+Tickless mode is an optimization mechanism to suspend the SysTick timer. In particular, this will put the system to uninterrupted sleep to achieve power saving when idle for multiple ticks. Target in tickless mode normally disables the SysTick, sets up wake-up timers and enters sleep mode when idle. When SysTick timer expires on its own or from an external event like interrupt, then it exits sleep mode and re-enables the SysTick.
+
+[Mbed OS sleep modes](../../../apis/power-management-sleep.html):
```

Comment:
An introduction to the Mbed OS sleep modes is missing.
For example, you could say that a further power optimisation is achieved by entering deep sleep when the system is idle.